src/Sparrow/Json/ArenaMemoryAllocator.cs

```diff
@@ -198,12 +189,23 @@ private void GrowArena(int requestedSize)
         private int GetPreferredSize(int requestedSize)
         {
             if (AvoidOverAllocation)
-                return Bits.NextPowerOf2(requestedSize);
+                return ApplyLimit(Bits.NextPowerOf2(requestedSize));
```

Comment:
We can remove ApplyLimit from here I think?

Reply:
Don't do any harm, and avoid needing to think why it isn't here.
Okay either way